### ISSUE TITLE
`templates` directory does not exists in npm distribution v0.0.4

### ISSUE THREAD

**Issue by nyarla** [r7kamura/fluct#1]
Installed by:

``` bash
$ npm install -g fluct
```

Run command:

``` bash
$ fluct new myapp
```

But I got this error:

```
events.js:141
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open '/home/nyarla/.anyenv/envs/ndenv/versions/iojs-v3.0.0/lib/node_modules/fluct/lib/../templates/package.json'
    at Error (native)
```

So, I think that this error happened by missing `templates` directory in npm distribution,
And this error maybe can fix by adding `templates` directory to package.json.

Regards.

**Comment by r7kamura** [comment]
@nyarla Thanks! It's my mistake. I just released v0.0.5 that includes templates dir in its package.

**Comment by nyarla** [comment]
@r7kamura  Thank you for fix!

:+1:
